py-polars/polars/series/series.py

```diff
@@ -5264,6 +5264,14 @@ def map_elements(
             This method is much slower than the native expressions API.
             Only use it if you cannot implement your logic otherwise.
 
+            For concreteness, suppose that the function is: `x ↦ sqrt(x)`.
+
+            For mapping elements of columns, consider:
+            `pl.col("colname").sqrt()`.
+
+            For mapping elements of inner lists, consider:
+            `pl.col("colname").list.eval(pl.element().sqrt())`.
```

Comment:
these are Series docs, so the example should use `s` instead of `pl.col('colname')`